app/code/core/Mage/GoogleAnalytics/etc/system.xml

```diff
@@ -57,6 +57,32 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </account>
+                        <user_id translate="label">
```

Comment:
```suggestion
                        <user_id translate="label comment">
```

Reply:
I don't really like the idea to track the customer id, anyway we should have a default value to false right?